### PR TITLE
Renamed the fields in the keyset Keys snapshot to make more sense

### PR DESCRIPTION
**Pull request by cody-constine-ttd**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.uid2</groupId>
     <artifactId>uid2-shared</artifactId>
-    <version>4.3.0-dddd3edffd</version>
+    <version>4.3.2-SNAPSHOT</version>
```
Comment (jon8787) [inline pom.xml:9]: I thought this gets committed as part of the pipeline

File: src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java [hunk 2]
```diff
@@ -7,28 +7,28 @@
 import java.util.*;
 
 public class KeysetKeyStoreSnapshot implements IKeysetKeyStore.IkeysetKeyStoreSnapshot {
-    private final HashMap<Integer, KeysetKey> keyMap;
-    private final HashMap<Integer, List<KeysetKey>> keysetMap;
-    private final List<KeysetKey> activeKeys;
+    private final HashMap<Integer, KeysetKey> keyIdToKeyMap;
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java:10]: keyIdToKeysetKey

File: src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java [hunk 3]
```diff
@@ -7,28 +7,28 @@
...
+    private final HashMap<Integer, List<KeysetKey>> keysetIdToKeyListMap;
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java:11]: keysetIdToKeysetKeyList

File: src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java [hunk 4]
```diff
@@ -7,28 +7,28 @@
...
+    private final List<KeysetKey> allKeys;
 
-    public KeysetKeyStoreSnapshot(HashMap<Integer, KeysetKey> keyMap, HashMap<Integer, List<KeysetKey>> keysetMap) {
-        this.keyMap = keyMap;
-        this.keysetMap = keysetMap;
-        this.activeKeys = new ArrayList<>(keyMap.values());
+    public KeysetKeyStoreSnapshot(HashMap<Integer, KeysetKey> keyIdToKeysetKey, HashMap<Integer, List<KeysetKey>> keysetIdToKeysetKeyList) {
+        this.keyIdToKeyMap = keyIdToKeysetKey;
+        this.keysetIdToKeyListMap = keysetIdToKeysetKeyList;
+        this.allKeys = new ArrayList<>(keyIdToKeysetKey.values());
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java:17]: btw why do we store this, is it too slow to call keyIdToKeysetKey.values() each time ?

File: src/main/java/com/uid2/shared/store/IKeysetKeyStore.java [hunk 5]
```diff
@@ -11,7 +11,7 @@ public interface IKeysetKeyStore {
 
 
     public interface IkeysetKeyStoreSnapshot {
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/IKeysetKeyStore.java:13]: why does this interface exist - can we remove it?

File: src/main/java/com/uid2/shared/store/parser/KeysetKeyParser.java [hunk 6]
```diff
@@ -19,8 +19,8 @@ public class KeysetKeyParser implements Parser<IKeysetKeyStore.IkeysetKeyStoreSn
     @Override
     public ParsingResult<IKeysetKeyStore.IkeysetKeyStoreSnapshot> deserialize(InputStream inputStream) throws IOException {
         JsonArray keysSpec = Utils.toJsonArray(inputStream);
-        final HashMap<Integer, KeysetKey> keyMap = new HashMap<>();
-        final HashMap<Integer, List<KeysetKey>> keysetMap = new HashMap<>();
+        final HashMap<Integer, KeysetKey> keyIdToKeyMap = new HashMap<>();
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/parser/KeysetKeyParser.java:22]: keyIdToKeysetKey

File: src/main/java/com/uid2/shared/store/parser/KeysetKeyParser.java [hunk 7]
```diff
@@ -19,8 +19,8 @@ public class KeysetKeyParser implements Parser<IKeysetKeyStore.IkeysetKeyStoreSn
...
+        final HashMap<Integer, List<KeysetKey>> keysetIdToKeyListMap = new HashMap<>();
```
Comment (jon8787) [inline src/main/java/com/uid2/shared/store/parser/KeysetKeyParser.java:23]: keysetIdToKeysetKeyList

**Review by jon8787 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (cody-constine-ttd) [reply pom.xml:9]: This was the pipeline 👍

File: src/main/java/com/uid2/shared/store/IKeysetKeyStore.java (hunk 5 above)
Comment (cody-constine-ttd) [reply src/main/java/com/uid2/shared/store/IKeysetKeyStore.java:13]: This is how all the other snapshots work

File: src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java (hunk 4 above)
Comment (cody-constine-ttd) [reply src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java:17]: Just copying how the encryption keys worked

File: src/main/java/com/uid2/shared/store/IKeysetKeyStore.java (hunk 5 above)
Comment (jon8787) [reply src/main/java/com/uid2/shared/store/IKeysetKeyStore.java:13]: bit harder to fix those ones though. We can fix this one and fix those later

Comment (jon8787) [reply src/main/java/com/uid2/shared/store/IKeysetKeyStore.java:13]: LGTM other than this nested interface IkeysetKeyStoreSnapshot. Can we remove this, replace all instances with KeysetKeyStoreSnapshot, and then it's good to merge

Comment (cody-constine-ttd) [reply src/main/java/com/uid2/shared/store/IKeysetKeyStore.java:13]: I don't think this work is really in scope for the this story, we should do it on another card

**Review by jon8787 — APPROVED** [review]

**Pull request merged**